Scripts/RemediationScripts/Remediate-EnableSSLDBForMySQLFlexibleServer.ps1

```diff
@@ -0,0 +1,817 @@
+<###
+# Overview:
+    This script is used to set SSL for Azure Database for MySQL flexible server in a Subscription.
+
+# Control ID:
+    Azure_DBForMySQLFlexibleServer_DP_Enable_SSL
+
+# Display Name:
+    SSL must be enabled for Azure database for MySQL flexible server
+
+# Prerequisites:    
+    Contributor or higher priviliged role on the Azure Database for MySQL flexible server(s) is required for remediation.
+
+# Steps performed by the script:
+    To remediate:
+        1. Validate and install the modules required to run the script and validate the user..
+        2. Get the list of Azure Database for MySQL flexible server(s) in a Subscription that have server parameter require_secure_transport not set as ON.
+        3. Back up details of Azure Database for MySQL flexible server(s) that are to be remediated.
+        4. Set SSL by updating server parameter require_secure_transport as ON for Azure Database for MySQL flexible server(s).
+
+    To roll back:
+        1. Validate and install the modules required to run the script and validate the user.
+        2. Get the list of Azure Database for MySQL flexible server(s) in a Subscription, the changes made to which previously, are to be rolled back.
```

Comment:
Get the list of Azure Database for MySQL flexible server(s) in a Subscription, the changes made to which previously, are to be rolled back -> Get the list of Azure Database for MySQL flexible server(s) in a Subscription, t**o which changes were made previously and are to be rolled back, from the backed-up data.**